### PR TITLE
Update RELEASE_NOTE.txt for Release 4.1.3

### PR DESCRIPTION
**Pull request by szab100**

### CODE REVIEW

File: RELEASE_NOTE.txt [hunk 1]
```diff
@@ -1,13 +1,53 @@
 Release Notes:
 ===============
 
-Google SAML Bridge for Windows
+SAML Bridge for Google Search Appliance
```
Comment (wiarlawd) [inline RELEASE_NOTE.txt]: Sorry, I misremembered the name fix. It's just dropping "Google", so "SAML Bridge for Windows".

File: RELEASE_NOTE.txt [hunk 2]
```diff
@@ -1,13 +1,53 @@
...
 ===============
 
-Google SAML Bridge for Windows
+SAML Bridge for Google Search Appliance
 
-This document contains the release notes for Google SAML Bridge for Windows.
+This document contains the release notes for SAML Bridge for Google Search Appliance.
 The following sections describe the release in detail and provide information that supplements the main documentation.
 
 Web Site: http://code.google.com/p/google-saml-bridge-for-windows/
```
Comment (wiarlawd) [inline RELEASE_NOTE.txt]: Change to https://github.com/googlegsa/samlbridge

File: RELEASE_NOTE.txt [hunk 3]
```diff
@@ -1,13 +1,53 @@
 Release Notes:
 ===============
 
-Google SAML Bridge for Windows
+SAML Bridge for Google Search Appliance
 
-This document contains the release notes for Google SAML Bridge for Windows.
+This document contains the release notes for SAML Bridge for Google Search Appliance.
```
Comment (wiarlawd) [inline RELEASE_NOTE.txt]: Here let's say "... for SAML Bridge for Windows, a part of the Google Search Appliance."

File: RELEASE_NOTE.txt [hunk 4]
```diff
@@ -1,13 +1,53 @@
...
+
+Introduction
+------------
+This is a patch release that adds a simple new feature requested since the previous release. Users of previous releases are encouraged to upgrade only if they need these new features.  Your feedback is important to us.
+
+Summary of Changes / New Features
+---------------------------------
+This patch contains fixes for the following issues in the connector:
+ 
+-- SAML Bridge supports multi-host GSA configuration (GSA^n) using assertion_consumer URL from the SAML request.
```
Comment (wiarlawd) [inline RELEASE_NOTE.txt]: This should include the issue number, as previous releases did.

File: RELEASE_NOTE.txt [hunk 5]
```diff
@@ -1,13 +1,53 @@
...
+-- SAML Bridge source code is updated to VS 2010+ format.
```
Comment (wiarlawd) [inline RELEASE_NOTE.txt]: This doesn't need to be included in the release notes.

File: RELEASE_NOTE.txt (hunk 1 above)
Comment (szab100) [inline RELEASE_NOTE.txt]: Done

File: RELEASE_NOTE.txt (hunk 3 above)
Comment (szab100) [inline RELEASE_NOTE.txt]: Done

File: RELEASE_NOTE.txt (hunk 2 above)
Comment (szab100) [inline RELEASE_NOTE.txt]: Done

File: RELEASE_NOTE.txt (hunk 4 above)
Comment (szab100) [inline RELEASE_NOTE.txt]: Done

File: RELEASE_NOTE.txt (hunk 5 above)
Comment (szab100) [inline RELEASE_NOTE.txt]: Done

File: RELEASE_NOTE.txt [hunk 6]
```diff
@@ -1,12 +1,51 @@
 Release Notes:
 ===============
 
-Google SAML Bridge for Windows
+SAML Bridge for Windows
 
-This document contains the release notes for Google SAML Bridge for Windows.
+This document contains the release notes for SAML Bridge for Windows, a part of the Google Search Appliance..
```
Comment (wiarlawd) [inline RELEASE_NOTE.txt]: Delete the extra period at the end.

**Pull request merged**